Sixteen750/src/main/java/org/firstinspires/ftc/sixteen750/subsystems/IntakeSubsystem.java

```diff
@@ -12,7 +12,9 @@
 @Configurable
 public class IntakeSubsystem {
 
-    public static double MOTOR_VELOCITY = 0.25; // 0.5 1.0
+    public static double INTAK_VELO = 0.5; // 0.5 1.0
```

Comment:
Same as before about comments regarding bot testing. Also: Spell it "INTAKE_VELO". It looks odd, and Android Studio will give us red squiggles 😒